pkg/api/schema/schema.proto

```diff
@@ -525,4 +566,33 @@ service ImmuService {
 	//			body: "*"
 	//		};
 	//	}
+	rpc CreateDatabase(Database) returns (CreateDatabaseReply) {
+		option (google.api.http) = {
+			post: "/v1/immurestproxy/createdatabase"
+			body: "*"
+		};
+	}
+	rpc UseDatabase(Database) returns (Error) {
```

Comment:
It seems that when I switch DB with `usedatabase` next SET operation are forwarded to default DB. I'm missing something?

Reply:
Yes, this was due to a bug in devmode handling